tests/saitests/sai_qos_tests.py

```diff
@@ -1714,13 +1714,24 @@ def runTest(self):
         tos |= ecn
         ttl = 64
         default_packet_length = 64
-        pkt = simple_tcp_packet(pktlen=default_packet_length,
-                                eth_dst=router_mac if router_mac != '' else dst_port_mac,
+        pkt_dst_mac = router_mac if router_mac != '' else dst_port_mac
+        pkt = simple_ip_packet(pktlen=default_packet_length,
```

Comment:
@abdosi TCP header doesn't factor in this test, it doesn't affect the test. Also confirmed with Wei Bai, ECN marking for IP header is what matters. @baiwei0427

Reply:
> prefer to keep tcp packets. Most of ASIC's does ECN Marking only for packets with TCP Header.
> Can we fix src port and dest port so to get deterministic behavior.

Actually RoCE (RDMA over Converged Ethernet) packets use UDP header. ECN field is in IP header rather than in TCP/UDP header.